mqtt/mqtt-broker-tests-util/src/packet.rs

```diff
@@ -0,0 +1,109 @@
+use std::{
```

Comment:
nit: let's name this module "packet_stream". I think just "packet" can be confusing.